buildscripts/make_dependencies.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Build protoc & netty
+set -ev
+
+# Make protoc
+pushd .
+cd /tmp
+git clone https://github.com/google/protobuf.git
```

Comment:
By switching to sudo-free we now get 2 cores

http://blog.travis-ci.com/2014-12-17-faster-builds-with-container-based-infrastructure/

will enhance the calm here